scripts/plugins/storage/sqlite/upgrade/72.sql

```diff
@@ -1,4 +1,5 @@
 ALTER TABLE fledge.users ADD COLUMN failed_attempts INTEGER DEFAULT 0;
-ALTER TABLE fledge.users ADD COLUMN block_until DATETIME DEFAULT "2024-01-01 10:10:10.55";
+ALTER TABLE fledge.users ADD COLUMN block_until DATETIME DEFAULT NULL;
+UPDATE fledge.users SET block_until=current_timestamp;
```

Comment:
Sqlite: CURRENT_TIMESTAMP is in GMT, not the timezone of the machine.

`UPDATE users SET block_until=(SELECT STRFTIME('%Y-%m-%d %H:%M:%f', 'NOW', 'localtime'))`

Reply:
Why are we not setting null? TS should be only if blocked.